src/eos-autoupdater.c

```diff
@@ -59,7 +59,7 @@ typedef enum _UpdateStep {
 #define SEC_PER_DAY (3600ll * 24)
 
 /* This file is touched whenever the updater starts */
-static const char *UPDATE_STAMP_DIR = LOCALSTATEDIR "/eos-updater";
+static const char *UPDATE_STAMP_DIR = SHAREDSTATEDIR "/eos-updater";
```

Comment:
I don't think this is right. The autoconf default for sharedstatedir is `$prefix/com` (http://git.savannah.gnu.org/gitweb/?p=autoconf.git;a=blob;f=lib/autoconf/general.m4;hb=HEAD#l587). Likewise, debhelper does no overriding of sharedstatedir (https://anonscm.debian.org/cgit/debhelper/debhelper.git/tree/Debian/Debhelper/Buildsystem/autoconf.pm#n29).

So, this does introduce a functional change moving the stamp directory from `/var/lib/eos-updater` to `/usr/com/eos-updater`.